draft-wang-ppm-differential-privacy.md

```diff
@@ -762,11 +757,12 @@ class DpPolicy:
 
 ## Executing DP Policies with VDAFs {#run-dp-policy-with-vdaf}
```

Comment:
Since the function below is renamed to `run_vdaf_with_dp_policy`, maybe make a pass to change it for all ? E.g. “Executing VDAFs with DP Policies {#run-vdaf-with-dp-policy}”.

Reply:
Done.